pallets/loans-ref/src/lib.rs

```diff
@@ -240,14 +240,14 @@ pub mod pallet {
 		_,
 		Blake2_128Concat,
 		PoolIdOf<T>,
-		BoundedVec<WriteOffState<T::Rate>, T::MaxWriteOffGroups>,
+		BoundedVec<WriteOffState<T::Rate>, T::MaxWriteOffPolicySize>,
 		ValueQuery,
 	>;
 
 	/// Stores the portfolio valuation associated to each pool
 	#[pallet::storage]
 	#[pallet::getter(fn portfolio_valuation)]
-	pub(crate) type LatestPortfolioValuations<T: Config> =
+	pub(crate) type LatestPortfolioValuation<T: Config> =
```

Comment:
I wonder if we should just change this to `PortfolioValuation`? A lot of storage values can change but don't include the latest prefix. E.g. `WriteOffPolicy` is also storing the latest policy.

Reply:
It makes sense!